docs-2.0/3.ngql-guide/6.funtions-and-expressions/5.case-expressions.md

```diff
@@ -0,0 +1,161 @@
+# CASE expressions
+
+The `CASE` expression uses conditions to filter the result of an nGQL query statement. It is used in the `YIELD` or `RETURN` clause. nGQL provides two forms of `CASE` expressions just like openCypher: the simple form and the generic form.
+
+The `CASE` expression goes through conditions and returns a result when the first condition is met. Then the `CASE` expression stops reading the conditions and returns the result. If no conditions are met, it returns the result in the `ELSE` clause. If there is no `ELSE` clause and no conditions are met, it returns `NULL`.
+
+The following graph is used for the examples in this topic.
+
+![Example graph for CASE expressions](https://docs-cdn.nebula-graph.com.cn/docs-2.0/3.ngql-guide/6.funtions-and-expressions/case-example.png)
+
+## The simple form of CASE expressions
+
+### Syntax
+
+```nGQL
+CASE <comparer>
+WHEN <value> THEN <result>
+[WHEN ...]
+[ELSE <default>]
+END
+```
+
+> **CAUTION:** Always remember to end a `CASE` expression with `END`.
+
+|Parameters|Description|
+|-|-|
+|`comparer`|A value or a valid expression that outputs a value. This value is used to compare with `value`.|
+|`value`|It will be compared with `comparer`. If they match, then this condition is met.|
+|`result`|It is returned by the `CASE` expression if `value` matches `comparer`.|
+|`default`|It is returned by the `CASE` expression if no conditions are met.|
+
+### Examples
+
+* Example 1:
+
+    ```nGQL
+    nebula> YIELD \
+         -> CASE 2+3 \
+         -> WHEN 4 THEN 0 \
+         -> WHEN 5 THEN 1 \
+         -> ELSE -1 \
+         -> END \
+         -> AS result;
```

Comment:
what happens if no AS here

Reply:
Try it out~
It outputs the query as the table title.